src/Analysis/Engine/Impl/KnownTypes.cs

```diff
@@ -147,27 +135,17 @@ class FallbackBuiltinModule : IBuiltinPythonModule, IPythonModule {
         }
 
         public IEnumerable<string> GetChildrenModules() => Enumerable.Empty<string>();
-        public IMember GetMember(IModuleContext context, string name) => null;
-        public IEnumerable<string> GetMemberNames(IModuleContext moduleContext) => Enumerable.Empty<string>();
         public void Imported(IModuleContext context) { }
     }
 
-    class FallbackBuiltinPythonType : IPythonType {
-        public FallbackBuiltinPythonType(IBuiltinPythonModule module, BuiltinTypeId typeId, string name = null) {
-            DeclaringModule = module;
-            Name = name ?? typeId.GetModuleName((DeclaringModule as FallbackBuiltinModule)?.LanguageVersion ?? PythonLanguageVersion.None);
+    class FallbackBuiltinPythonType : AstPythonType {
+        public FallbackBuiltinPythonType(IBuiltinPythonModule declaringModule, BuiltinTypeId typeId, string name = null) :
```

Comment:
`FallbackBuiltinPythonType` constructor is called only in two places, in both `declaringModule` is `FallbackBuiltinModule` and its first constructor parameter, and `name` is always `null`